sunshine/confighttp.cpp

```diff
@@ -335,6 +362,29 @@ void getConfig(resp_https_t response, req_https_t request) {
   }
 }
 
+void getPlainPassword(resp_https_t response, req_https_t request) {
```

Comment:
Again, we could just have the welcome page generate the first username/password pair, this prevents the need for a getPlainPassword method